web/pdf_rendering_queue.js

```diff
@@ -164,9 +164,14 @@ class PDFRenderingQueue {
         break;
       case RenderingStates.INITIAL:
         this.highestPriorityPage = view.renderingId;
-        view.draw().finally(() => {
-          this.renderHighestPriority();
-        });
+        view
+          .draw()
+          .finally(() => {
+            this.renderHighestPriority();
+          })
+          .catch(() => {
+            /* Avoid "Uncaught promise" messages in the console. */
```

Comment:
Can we either log the error or remove the catch? Empty catch statements can make debugging very painful.

Reply:
Fair points, any errors here are now logged (such that there's no "Uncaught promise" warnings) and a unit-test were added.